docs/install/integrations.md

```diff
@@ -169,29 +177,36 @@ dependencies {
     // ktlint(project(":custom-ktlint-ruleset")) // in case of custom ruleset
 }
 
-val outputDir = "${project.buildDir}/reports/ktlint/"
-val inputFiles = project.fileTree(mapOf("dir" to "src", "include" to "**/*.kt"))
-
-val ktlintCheck by tasks.creating(JavaExec::class) {
-    inputs.files(inputFiles)
-    outputs.dir(outputDir)
-
-    description = "Check Kotlin code style."
+val ktlintCheck by tasks.registering(JavaExec::class) {
+    group = LifecycleBasePlugin.VERIFICATION_GROUP
+    description = "Check Kotlin code style"
     classpath = ktlint
     mainClass.set("com.pinterest.ktlint.Main")
     // see https://pinterest.github.io/ktlint/install/cli/#command-line-usage for more information
-    args = listOf("src/**/*.kt")
+    args(
+        "**/src/**/*.kt",
+        "**.kts",
+        "!**/build/**",
```

Comment:
This set of patterns is used on multiple places now. It might be better to extract this to a shared constant.

Reply:
Yeah, but they are examples in docs, might be better not to reuse.

Reply:
Indeed, I did not realize that.